blueocean-jira/pom.xml

```diff
@@ -29,6 +29,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jira</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-api</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
just building this project on the `master` branch I did not even see `slf4j` being bundled in the hpi - so it this at all needed?

Reply:
fixed